python/cudf/cudf/utils/ioutils.py

```diff
@@ -229,7 +229,7 @@
     File path or Root Directory path. Will be used as Root Directory path
     while writing a partitioned dataset. Use list of str with partition_offsets
     to write parts of the dataframe to different files.
-compression : {{'snappy', 'ZSTD', None}}, default 'snappy'
+compression : {{'snappy', 'ZSTD', 'lz4', None}}, default 'snappy'
```

Comment:
PyArrow appears to support a different set of values (the casing is different for the values in common, the ordering is different, and `'NONE'` is different than `None`):

> Valid values: {‘NONE’, ‘SNAPPY’, ‘GZIP’, ‘BROTLI’, ‘LZ4’, ‘ZSTD’}.

https://arrow.apache.org/docs/python/generated/pyarrow.parquet.write_table.html

Reply:
We don't support all compression algorithms that pyarrow does, the list is correct. For the rest, https://github.com/rapidsai/cudf/pull/14906#discussion_r1486691174 Applies here as well.